src/libcore/convert.rs

```diff
@@ -178,14 +178,14 @@ pub trait Into<T>: Sized {
 /// ```
 /// # Generic impls
 ///
-/// - `From<T> for U` implies `[Into<U>] for T`
+/// - `From<T> for U` implies [`Into<U>`]` for T`
```

Comment:
You can see how this renders [here](https://doc.rust-lang.org/nightly/std/convert/trait.From.html). Basically you can't have links inside code blocks.

Before:
 - `From<T> for U` implies `[Into<U>] for T`
[Into<U>]: trait.Into.html

After:
 - `From<T> for U` implies [`Into<U>`]` for T`
[`Into<U>`]: trait.Into.html